executors.go

```diff
@@ -286,6 +286,7 @@ func (c *Connection) Create(model interface{}, excludeColumns ...string) error {
 					}
 					stm := after[index].AfterProcess()
 					if c.TX != nil && !stm.Empty() {
+						log(logging.SQL, stm.Statement, stm.Args)
```

Comment:
please remove the last commit for these two lines. The executing code was improved, and the logging also already modified.

Reply:
done.